changelog.md

```diff
@@ -3,3 +3,7 @@
 ## [1.0.0] - 2019-03-11
 - Rename package from `digipolis-login` to `auth`
 - First release to npmjs
+
+## [1.1.0] - 2019-06-05
```

Comment:
It's good practice to reverse the order of the changelog. This way the most relevant changes are at the top
Can you also add the link to the tag? This might not yet exist before this PR is merged but the url can be predicted.